biggerquery/deployment.py

```diff
@@ -19,7 +19,7 @@ def job_callable(**kwargs):
     return job_callable
 
 
-def workflow_to_dag(workflow, start_from, dag_id):
+def workflow_to_dag(workflow, start_from, dag_id, **dag_kwargs):
```

Comment:
Adding the kwargs here is not enough, you need to allow the user to pass the kwargs to the Workflow class and then use the kwargs inside workflow_to_dag

Reply:
done

Reply:
Did you test it using Cloud Composer?

Reply:
not yet, today will be tested

Reply:
after that it will be ready to merge I guess